.circleci/config.yml

```diff
@@ -16,6 +16,9 @@ executors:
           POSTGRES_USER: postgres
           POSTGRES_PASSWORD: secretpass
           POSTGRES_DB: ttasmarthub
+  docker-python-executor:
+    docker:
+      - image: cimg/python:3.9.17
```

Comment:
is there a reason 3.9.17 is used over 3.9.18 or even 3.11.5? What python version is in the build pack, as I think they should match.

Reply:
https://github.com/cloudfoundry/python-buildpack/releases
It looks like the latest python build pack no longer supports 3.9.17. So 3.9.18 or 3.10.13 or 3.11.5 should be used.

I think 3.10.13 looks to be the newest stable version. as 3.11.5 is still in bug fix according to https://devguide.python.org/versions/

Reply:
I bumped from 3.9.17 to 3.9.18 b58ac61 for this exact reason (CF deprecation) but forgot to update the version here as well

reconciled here: 2f1d84d19